client/my-sites/comments/comment/comment-content.jsx

```diff
@@ -127,14 +125,11 @@ const mapStateToProps = ( state, { commentId } ) => {
 	const parentCommentId = get( comment, 'parent.ID', 0 );
 	const parentCommentContent = decodeEntities( stripHTML( get( parentComment, 'content' ) ) );
 
-	const parentCommentUrl = isJetpack
-		? get( parentComment, 'URL' )
```

Comment:
This is a nice fix and I've verified it works well for nested comments in Jetpack sites, but I wonder why it ended up being in this PR 😅  Maybe we should move it to another one?

Reply:
It's true that it's an independent fix that fixes a regression introduced in this commit:

https://github.com/Automattic/wp-calypso/commit/3d085b0bb000a066ae0ffe7b3df3a966411918fb#diff-1d6d1a285110a4bfa3111ed34eb5c1966cd7905b18a4311d5e04da5c6c8714b0L132

Note that the `comments/management/comment-view` feature flag is always true, so the `else` branches, including he `isJetpack` one, should be removed. But the `isJetpack` one wasn't removed.

I'll move it to a separate PR.

Reply:
👍  Thanks, please ping me for a review

Reply:
Done in #53219.